BLSQ/iaso
# WC2-299: Create UI to launch duplicate analyses

Explain what problem this PR is resolving
- There is only relaunch analysis, there was not launch analysis
- This ticket is for adding a new button for launching a new duplicate analysis
Related JIRA tickets : [WC2-299](https://bluesquare.atlassian.net/browse/WC2-299)

## Self proofreading checklist


## Changes
- Add a new button
- Add a modal when the button is clicked

## How to test
- Go on beneficiaries menu and click on the duplicate one submenu
- Click on the launch analysis button the top-right side
- Select an entity type and an algorithm
- and confirm
## Print screen / video
[Screencast from 2023-11-30 15-40-17.webm](https://github.com/BLSQ/iaso/assets/19631540/5fdcd2f6-ba41-4cc1-8721-b6f5ddc2d01d)



[WC2-299]: https://bluesquare.atlassian.net/browse/WC2-299?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
